app/views/course/assessment/question/programming/_form.json.jbuilder

```diff
@@ -1,10 +1,18 @@
 # frozen_string_literal: true
 json.partial! 'course/assessment/question/skills', course: course
 
-json.languages Coursemology::Polyglot::Language.all.order(:name) do |language|
-  json.id language.id
-  json.name language.name
-  json.editorMode language.ace_mode
+json.languages Coursemology::Polyglot::Language.all.order(:weight) do |language|
+  if language.enabled
```

Comment:
instead of using if/elsif and then repeating the same pattern all over again (those `json.id`, `json.name`, `json.editorMode` are all same), why not using the following pattern

```
next unless language.enabled || @programming_question.language_id == language.id

json.id language.id
...
```

for `json.disabled`, it should be `!language.enabled`